tools/run-conftest.sh

```diff
@@ -5,9 +5,9 @@ set -e
 set -o errexit
 set -o pipefail
 
-DO_DEPENDENCY_UPDATE=0
 GIT_REPO_ROOT=$(git rev-parse --show-toplevel || echo -n '/')
-POLICIES_PATH=$GIT_REPO_ROOT/operations/helm/policies
+POLICIES_PATH="$GIT_REPO_ROOT/operations/helm/policies"
+GENERATED_DIR="$GIT_REPO_ROOT/operations/helm/tests/intermediate"
```

Comment:
This is being shared by the two scripts implicitly. Can you make it an argument to both and have it hardcoded in the makefile? Or some other way of making sure they use the same script

Reply:
fixed